src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -250,7 +251,7 @@ private void bindLauncherControls() {
 		// 				s.launcherSubsystem.runEnd(
 		// 						s.launcherSubsystem::launch, s.launcherSubsystem::stopLauncher));
 
-		driveController.b().onTrue(new InstantCommand(() -> s.launcherSubsystem.launch(4000)));
+		driveController.b().onTrue(new InstantCommand(() -> s.launcherSubsystem.launch(6500)));
```

Comment:
Note that driveController.b is mapped to both this & the driveIntakeStopButton. Doing both those actions on 'b' seems reasonable, but would be nice to only map the button in one place.

Reply:
Oh, and the bump to 6500 here is specifically for use when launching notes cross-field.